tla-io/src/main/scala/at/forsyte/apalache/tla/imp/StandardLibrary.scala

```diff
@@ -79,6 +81,8 @@ object StandardLibrary {
     Map(
         "TLC.tla" -> "__rewire_tlc_in_apalache.tla",
         "Sequences.tla" -> "__rewire_sequences_in_apalache.tla",
+        // will be enabled later
+        //        "SequencesExt.tla" -> "__rewire_sequences_ext_in_apalache.tla",
```

Comment:
Can you add to this what "later" means, e.g. what the blocking issue is?

Reply:
in the next PR :)

Reply:
Fair enough